### PR TITLE
Lab 8

### PR DESCRIPTION
**Pull request by ShawnCarpenter**

### CODE REVIEW

File: app/cart.js [hunk 1]
```diff
@@ -1,24 +1,48 @@
 //import data and scripts
-
```
Comment (dakotadruley) [inline app/cart.js:2]: to get into the habit of writing clean code, make sure to keep all imports at the top with no extra spaces.

File: index.html [hunk 2]
```diff
@@ -27,9 +27,6 @@ <h1>Tony's House of Tapes</h1>
             <h3>We have your solution</h3>
             <h1>Tapes!</h1>
         </section>
-        <section>
-          <img src="images/1024px-Compactcassette.jpg" alt="">
```
Comment (dakotadruley) [inline index.html:31]: for accessibility, you can always include an "alt" by defaulting to the name of the image or a description as a string which a screen reader would pick up.

Comment (ShawnCarpenter) [reply index.html:31]: Thanks, I'll keep that in mind, are there any good resources on accessibility to look at?

File: app/cart.js (hunk 1 above)
Comment (ShawnCarpenter) [reply app/cart.js:2]: Thanks, I've fixed it.
Looking at it I think my reasoning was to separate the data imports from the function imports, is that not necessary?
Or should I do it like this:

//import static data
import tapes from './tapes.js';

//import functions
import { renderLineItem } from './render-line-item.js';

**Pull request merged**